jetstream/analysis.py

```diff
@@ -204,7 +204,30 @@ def _calculate_statistics(self, metrics_table: str, period: AnalysisPeriod):
         for m in self.config.metrics[period]:
             results += m.run(metrics_data, reference_branch).to_dict()["data"]
 
-        results += Count().transform(metrics_data, "*", "control").to_dict()["data"]
+        counts = Count().transform(metrics_data, "*", "*").to_dict()["data"]
+        results += counts
+
+        # add count=0 row to statistics table for missing branches
+        missing_counts = StatisticResultCollection(
+            [
+                StatisticResult(
+                    metric="identity",
+                    statistic="count",
+                    parameter=None,
+                    branch=b.slug,
+                    comparison=None,
+                    comparison_to_branch=None,
+                    ci_width=None,
+                    point=0,
+                    lower=None,
+                    upper=None,
+                )
+                for b in self.config.experiment.branches
+                if not any(c["branch"] == b.slug for c in counts)
```

Comment:
this reads a little better to me but I'm happy with this as-is

```suggestion
               if b.slug not in {c["branch"] for c in counts}
```